up42/order.py

```diff
@@ -0,0 +1,178 @@
+from time import sleep
+from typing import Dict, List, Optional
+
+from up42.auth import Auth
+from up42.asset import Asset
+from up42.tools import Tools
+from up42.utils import (
+    get_logger,
+)
+
+logger = get_logger(__name__)
+
+DATA_PROVIDERS = ["oneatlas"]
+
+
+class Order(Tools):
```

Comment:
Semantically it sounds odd that an Order "is a" tool (which is the meaning of the inheritance relation).

Reply:
Thanks! You're absolutely right, @chrieke is in the process of deprecating the Tools object, but we encountered some mypy problems once we remove the inheritance. For some reason, once the class has no parent, we get a lot of mypy errors (I believe this is due to the fact that the method `Auth._request` has an undefined type - can be Response, json or text). We want to refactor this method and at that point we could also remove this unnatural inheritances.